07-http_gordle/6_6_query_param/internal/handlers/newgame/doc.go

```diff
@@ -1,3 +1,3 @@
 // Package newgame is the implementation of the http handler for the new game endpoint.
-// NewGame can only Add a game in the game repository.
+// NewGame can only Create a game in the game repository.
```

Comment:
the interface is `gameAdder` - we might want to change that too.

Reply:
Keep Add everywhere.